packages/create-catalyst/README.md

```diff
@@ -34,3 +34,46 @@ pnpm create @bigcommerce/catalyst@latest init
 ```sh
 yarn create @bigcommerce/catalyst@latest init
 ```
+
+### Develop a native integration for a new Catalyst project
+
+> **Note:** Currently there aren't any anticipated use cases that require the usage of the `integration` command outside of the Catalyst monorepo. Therefore, `pnpm` is assumed to be the most common package manager that this command will be executed with.
+
+```sh
+pnpm create @bigcommerce/catalyst@latest integration
+```
+
+#### How to develop a native integration for new Catalyst projects
+
+If you are interested in developing a native integration for Catalyst, you can use the `integration` command documented above.
+
+First, you'll need to write the code that makes your integration work with Catalyst. Begin by forking the [`bigcommerce/catalyst` repository](https://github.com/bigcommerce/catalyst), clone the fork locally, and [follow the steps to get started with local monorepo development](https://www.catalyst.dev/docs/monorepo). Be sure to add the original `bigcommerce/catalyst` repository as a remote (often named `upstream`) so that you can pull in latest updates pushed to `bigcommerce/catalyst:main`.
+
+Next, create a branch off of `main` and name it whatever you'd like. This branch is where you'll be building your integration into Catalyst.
```

Comment:
Might need to re-think this, since Catalyst projects are created based on the latest `catalyst-core` release, not main. Maybe we should create integration branches off of the latest `catalyst-core` release tag? And "updating" integrations is as straightforward as rebasing on top of the latest release tag?